script.module.enum34/addon.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.enum34" version="1.1.6" name="enum" provider-name="jmooremcc">
```

Comment:
provider name should be Ethan Furman

Reply:
@Lunatixz If you modified the module to work within Kodi please use your name as the author (addons.xml). I modified the code per the code reviews. I have no problem restoring the original author as the provider.

Reply:
Did you actually modified the code?

Reply:
On July 3rd I implemented these changes:
https://github.com/xbmc/repo-scripts/pull/1079#issuecomment-508296867

Basically, I inserted calls to the built-in list() where it was indicated by the code review.
After I did this, the code passed the code review.

I'm just trying to follow the advice I'm being given and don't have a problem with changing the provider-name back to the original author.

Thanks

Reply:
Please do add any custom changes, keep it as it is. Also CI didn't fail because of not using list for sure.

Reply:
Keeping the changes
I changed the provider back to the original author

Reply:
Sorry my above comments was wrong, do **not** add any custom changes.